react-components/src/apps/quickLaunch/CreateQuickLaunchDialog.js

```diff
@@ -68,18 +64,6 @@ function CreateQuickLaunchDialog(props) {
                                 component={FormTextField}
                             />
                             <ErrorMessage name="name" component="div" />
-                            <Field
-                                id={build(
-                                    baseDebugId,
-                                    ids.QUICK_LAUNCH.description
-                                )}
-                                name="description"
-                                label={getMessage("descriptionLabel")}
-                                required={false}
-                                margin="dense"
-                                component={FormMultilineTextField}
-                            />
-                            <ErrorMessage name="description" component="div" />
```

Comment:
Currently, there is no UI to display the `description` associated with a `Quick Launch`. Removing `description` field for the first pass of `Quick Launch`. I will add them back when we start displaying stats for `Quick Launch`.